apps/frontend/src/libs/components/sidebar/styles.module.css

```diff
@@ -0,0 +1,15 @@
+.sidebar {
+	display: flex;
+	flex: 20%;
```

Comment:
Not sure what this is for. As I see from layout needed width can be achieved by paddings
```suggestion
```